examples/tutorial/jupyter/execution/pandas_on_ray/cluster/exercise_5.md

```diff
@@ -0,0 +1,124 @@
+![LOGO](../../../img/MODIN_ver2_hrz.png)
```

Comment:
Can we avoid text duplication we have here and in rst file? Can we just refer to rst doc here?